src/patternfly/components/ChipGroup/chip-group.scss

```diff
@@ -25,13 +25,15 @@
   --pf-c-chip-group--c-chip--MarginRight: var(--pf-global--spacer--xs);
   --pf-c-chip-group--c-chip--MarginBottom: var(--pf-global--spacer--xs);
 
+  // Close
+  --pf-c-chip-group--button--MarginLeft: var(--pf-global--spacer--xs);
```

Comment:
change var: `--pf-c-chip-group--button--MarginLeft` to `--pf-c-chip-group--c-button--MarginLeft`